titus_isolate/monitor/workload_monitor_manager.py

```diff
@@ -45,23 +46,25 @@ def report_metrics(self, tags):
             log.debug("Not reporting metrics because there's no workload manager available yet.")
             return
 
-        static_pool_cpu_usage = self.__get_pool_usage(STATIC)
-        burst_pool_cpu_usage = self.__get_pool_usage(BURST)
+        usage = self.get_cpu_usage(60, 60)
```

Comment:
@rostyboost : This is fine for the efficiency metrics stuff, but we'll want something else for the `cpu_usage` metrics.